ext/js/display/search-display-controller.js

```diff
@@ -158,7 +158,7 @@ export class SearchDisplayController {
         if (
             activeElement !== this._queryInput &&
             !this._isElementInput(activeElement) &&
-            !e.ctrlKey &&
+            (!e.ctrlKey || e.key === 'Backspace') &&
```

Comment:
I'm not following this. If `e.ctrlKey` and `e.key === 'Backspace'` were both true, then this would be (!true || true)

Reply:
```suggestion
            !(e.ctrlKey && e.key === 'Backspace') &&
``` i think this is what you intended?

Reply:
Did you mean `(e.ctrlKey && e.key === 'Backspace')` ? But regardless it does the same thing. 

`(!true || true)` = `true`
`(true && true)` = `true`

And if you actually did mean to put that ! there... `!(true && true)` = `false`

Reply:
Actually looking at the entire if statement instead of this little chunk of it, none of these suggestions make sense and they just break the thing.

Reply:
Adding proposal that doesn't work. This would break existing behavior:

```js
const isIgnoreFocusKey = e.ctrlKey || e.metaKey || e.altKey || e.key === ' ';
const isTriggerFocusKey = (e.ctrlKey && e.key === 'Backspace') || (e.key.length === 1 && e.key === 'Backspace')

if (activeElement !== this._queryInput && !this._isElementInput(activeElement) && (!isIgnoreFocusKey || isTriggerFocusKey)) { 
  ...
}
```